i18n/messages.json

```
@@ -917,6 +917,9 @@
   "tasks.viewGraph": "View Graph",
   "tasks.delegate": "Delegate",
   "tasks.reassign": "Reassign",
+  "tasks.submit.error": "An error occurred while submitting the task.",
+  "tasks.submit.error.alreadyFinished": "The task you tried to submit was already finished by other user.",
+  "tasks.submit.error.noPermissions": "You don't have the permissions needed to process this task.",
```

Perhaps something shorter and with the same meaning like:
```suggestion
  "tasks.submit.error.noPermissions": "You don't have permissions to process this task.",
```

---

Thanks!